packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -368,25 +366,25 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
   /// To control what kind of [ScrollPosition] is created for a [Scrollable],
   /// provide it with custom [ScrollController] that creates the appropriate
   /// [ScrollPosition] in its [ScrollController.createScrollPosition] method.
-  ScrollPosition get position => _position;
-  ScrollPosition _position;
+  ScrollPosition? get position => _position;
```

Comment:
can we make this non-nullable? (_position would still need to be nullable, and updatePosition would need to be updated to refer to _position instead in some places)

Reply:
Done